<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="HVTNDAL0215" failures="2" tests="4" name="Test" time="0.744" errors="0" timestamp="01 Nov 2022 16:18:53 GMT">
  <testcase classname="allreTestCases.Testingg" name="case1" time="0.327">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at allreTestCases.Testingg.case1(Testingg.java:41)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- case1 -->
  <testcase classname="allreTestCases.Testingg" name="case2" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at allreTestCases.Testingg.case2(Testingg.java:50)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- case2 -->
  <testcase classname="allreTestCases.Testingg" name="case3" time="0.022"/>
  <testcase classname="allreTestCases.Testingg" name="case4" time="0.016"/>
</testsuite> <!-- Test -->
